mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -15,28 +15,45 @@
 package system
```

Comment:
Does `system update` need to be changed as well or is `update` completely reusing `start` functions?